packages/core/core-flows/src/order/workflows/confirm-return-request.ts

```diff
@@ -0,0 +1,92 @@
+import { OrderChangeDTO, OrderDTO, ReturnDTO } from "@medusajs/types"
+import { ChangeActionType } from "@medusajs/utils"
+import {
+  createStep,
+  createWorkflow,
+  transform,
+  WorkflowData,
+} from "@medusajs/workflows-sdk"
+import { useRemoteQueryStep } from "../../common"
+import { confirmOrderChanges } from "../steps/confirm-order-changes"
+import { createReturnItems } from "../steps/create-return-items"
+import {
+  throwIfIsCancelled,
+  throwIfOrderChangeIsNotActive,
+} from "../utils/order-validation"
+
+const validationStep = createStep(
+  "validate-create-return-shipping-method",
+  async function ({
+    order,
+    orderChanges,
+    orderReturn,
+  }: {
+    order: OrderDTO
+    orderReturn: ReturnDTO
+    orderChanges: OrderChangeDTO[]
+  }) {
+    throwIfIsCancelled(order, "Order")
+    throwIfIsCancelled(orderReturn, "Return")
+    throwIfOrderChangeIsNotActive({ orderChange: orderChanges })
+  }
+)
+
+export const confirmReturnRequestWorkflowId = "confirm-return-request"
+export const confirmReturnRequestWorkflow = createWorkflow(
+  confirmReturnRequestWorkflowId,
+  function (input: { return_id: string }): WorkflowData<string> {
```

Comment:
self-review: Create type + add `no_notification` and ensure it is correctly passed through